hedyorg/hedy
# 🧪 Refactor cypress tests (part 3)

Refactors the entire `for-teacher folder` for https://github.com/hedyorg/hedy/issues/4027 :
- `customize-adventure folder` only data-cy and .it() refactoring

**How to test**
I think the best way is to click around the for-teacher page tests and see if they are structured better, readabilty improved and if they cover testing the whole for-teacher page
